builtin/providers/google/resource_google_project_services.go

```diff
@@ -160,7 +168,9 @@ func getApiServices(pid string, config *Config) ([]string, error) {
 		return apiServices, err
 	}
 	for _, v := range svcResp.Services {
-		apiServices = append(apiServices, v.ServiceName)
+		if _, ok := ignore[v.ServiceName]; !ok {
+			apiServices = append(apiServices, v.ServiceName)
+		}
```

Comment:
We talked about this, and there was trouble with the count part of the Set and DiffSuppressFunc. I need to ask about this.